requirements-in.txt

```diff
@@ -1,7 +1,7 @@
 numpy==1.16.6
 scipy==1.4.1
 pillow
-opencv-python
+opencv-python-headless
```

Comment:
Actually, how about removing opencv-python, webapp, add headless to the copied version of req-in.txt with sed? Extra req.txt will be a pain to maintain

Reply:
I prefer a static file added into git instead of generating it dynamically, this will make things clear. You can imitate ./deploy/AidLux/requirements_generator. It's a ton of shitcode but do the trick.
FYI, AidLux is an terminal emulator on Android like termux, with pre-installed machine learning packages.